go.mod

```diff
@@ -1,16 +1,14 @@
 module github.com/prometheus/pushgateway
 
-go 1.23
-
-toolchain go1.23.0
+go 1.24
```

Comment:
```suggestion
go 1.23.0
```

Reply:
Hmm, I could have sworn this didn't work before, but now it does.